src/helm/benchmark/static/schema_mmlu.yaml

```diff
@@ -369,6 +369,7 @@ perturbations: []
 metric_groups:
   - name: accuracy
     display_name: Accuracy
+    hide_win_rates: true
```

Comment:
Do these need to be propagated to the other schema files?

Reply:
We could add `hide_win_rates: true` for General Information, but it doesn't seem too important.